pkg/deployment/agency/state.go

```diff
@@ -0,0 +1,117 @@
+//
+// DISCLAIMER
+//
+// Copyright 2016-2021 ArangoDB GmbH, Cologne, Germany
```

Comment:
why 2016. it is a new file and should start with 2021

Reply:
We use everywhere licenses like this